src/mero_wrk_tcp_binary.erl

```diff
@@ -147,8 +147,6 @@ transaction(Client, async_mset, [KVECs]) ->
     case async_mset(Client, KVECs) of
         {error, Reason} ->
             {error, Reason};
-        {ok, {error, Reason}} ->
```

Comment:
you sure this is not an actual case?

Reply:
Yeah… see [here](https://github.com/AdRoll/mero/blob/f3e1a4d7c613fabcfc2eab99940b26dbf94b6ba8/src/mero_wrk_tcp_binary.erl#L452). Only `ok` is acceptable.